src/lib/translators/mongo.ts

```diff
@@ -79,6 +79,19 @@ function transformReplace(step: ReplaceStep): MongoStep {
   };
 }
 
+/** transform a 'sort' step into corresponding mongo steps */
+function transformSort(step: SortStep): MongoStep {
+  let sortMongo: PropMap<number> = {};
```

Comment:
You should use `const` instead of `let` here since the `sortMongo` identifier (vs. its value) is not modified in the function.

```suggestion
   const sortMongo: PropMap<number> = {};
```